### PR TITLE
refactor: filesystem and styles restructure. added design system

### PR DESCRIPTION
**Pull request by falconx**

### CODE REVIEW

File: packages/ui/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "@toiletmap/api-client": "2.9.0",
     "@toiletmap/leaflet-control-geocoder": "2.9.0",
     "@toiletmap/opening-hours": "2.9.0",
+    "classnames": "^2.2.6",
```
Comment (marxian) [inline packages/ui/package.json]: pinned versions please!

File: packages/ui/src/design-system/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "styleguide",
+  "private": true,
+  "description": "Gatsby example site demoing living-style-guide",
+  "author": "scott.eckenthal@gmail.com",
```
Comment (marxian) [inline packages/ui/src/design-system/package.json]: Needs correcting

File: packages/ui/src/design-system/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "dependencies": {
+    "app-root-dir": "1.0.2",
+    "gatsby": "2.0.0",
+    "gatsby-source-filesystem": "2.0.1",
+    "gatsby-transformer-react-docgen": "2.1.1",
+    "gatsby-transformer-remark": "2.1.1",
+    "glamor": "2.20.40",
+    "html-to-react": "1.3.1",
+    "lodash": "^4.17.11",
```
Comment (marxian) [inline packages/ui/src/design-system/package.json]: pinned deps please

File: packages/ui/src/design-system/package.json (hunk 2 above)
Comment (falconx) [reply packages/ui/src/design-system/package.json]: I changed it a bunch of times and kept getting re-written :worried: but I will take another look.

**Pull request closed without merging**